src/index.ts

```diff
@@ -2,14 +2,17 @@ import { DOMWrapper } from './domWrapper'
 import { VueWrapper } from './vueWrapper'
 import BaseWrapper from './baseWrapper'
 import { mount, shallowMount } from './mount'
-import { renderToString } from './renderToString'
+import { renderToString as _renderToString } from './renderToString'
 import { MountingOptions } from './types'
 import { RouterLinkStub } from './components/RouterLinkStub'
 import { createWrapperError } from './errorWrapper'
 import { config } from './config'
 import { flushPromises } from './utils/flushPromises'
 import { enableAutoUnmount, disableAutoUnmount } from './utils/autoUnmount'
 
+// is __SSR__ avaialble? If so, preferable to use that?
```

Comment:
In Vue Core __SSR__ defined in rollup config https://github.com/vuejs/core/blob/da4a4fb5e8eee3c6d31f24ebd79a9d0feca56cb2/rollup.config.js#L148

Reply:
I don't know what's best. Maybe a variable defined in the rollup config would make more sense.

@lmiller1990 I think you were the one who set up the build config. Do you have an opinion on this?

Reply:
I'd expect we want to mirror what core does where possible!

Reply:
Either way, if it works, and we've got a test, happy to move forward with this. Did you want to make a change @Sandbagger? Any opinion @cexbrayat ?

Reply:
Thanks chaps. I'll give rollup config variable a go.